src/parser/maps.py

```diff
@@ -20,6 +20,14 @@ def get_target_type(value):
     return TARGET_TYPE_MAP.get(value)
 
 
+SOUL_UNLOCK_MAP = {
+    'm_unRequiredGold': 'RequiredGold',
```

Comment:
Is there a good reason to not call this souls?

Reply:
Changed to RequiredSouls on backend and frontend